components/camel-debezium/src/main/docs/debezium-component.adoc

```diff
@@ -0,0 +1,217 @@
+[[debezium-component]]
+= Debezium Component
+
+*Available as of Camel version 3.0*
+
+The Debezium component is wrapper around https://debezium.io/[Debezium] using https://debezium.io/docs/embedded/[Debezium Embedded], which enabled Change Data Capture from various databases without the need of Kafka or Kafka Connect.
+
+*Note on handling failures:* Per https://debezium.io/docs/embedded/#handling_failures[Debezium Embedded Engine] documentation, the engines is actively recording source offsets and periodically flushes these offsets to a persistent storage, so when the application is restarted or crashed, the engine will resume from the last recorded offset.
+Thus, at normal operation, your downstream routes will receive exactly once event, however in case of application crash (not having a graceful shutdown), the application will resume from the last recorded offset,
+which may result of receiving duplicate events immediately after the restart. Therefore, your downstream routes should be tolerant enough of such case and deduplicate events if needed.
+
+Maven users will need to add the following dependency to their `pom.xml`
+for this component.
+
+[source,xml]
+------------------------------------------------------------
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-debezium</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+------------------------------------------------------------
+
+== URI format
+
+[source,java]
+---------------------------
+debezium:connector-type[?options]
+
+---------------------------
+
+== Supported Debezium Connectors
+- https://debezium.io/docs/connectors/mysql/[MySql].
```

Comment:
Ok, so if you *only* support MySQL for now, then the fixed dependency would be ok. I'd then add a sentence here that other connectors are not supported yet.

Reply:
Added